index.html

```diff
@@ -0,0 +1,523 @@
+          />
+          <h4>Behind the Bouquet: Our Sourcing Story</h4>
+          <p>
+            Take a peek into how we choose responsible growers and why local
+            flowers make all the difference.
+            <a class="link-news" href="#">Read more</a>
+          </p>
+        </div>
+        <div class="third-news news-block">
+          <img
+            class="news-image-big"
+            src="./assets/ruslan-aiginin-vGw2uwHcnU0-unsplash.jpg"
+            alt=""
+          />
+          <h4>Seasonal Spotlight: Autumn Blooms</h4>
+          <p>
+            Discover which locally grown flowers shine in autumn and how to
+            style them for a cozy, seasonal touch at home.<br /><br />
+            Think warm colors, rich textures, and bouquets that mirror the
+            changing leaves outside your window. From rustic dahlias to elegant
+            chrysanthemums, autumn flowers bring natural artistry to your table.
+            We'll show you how to pair them with the right vases and accents to
+            create effortless seasonal charm.
+            <a class="link-news" href="#">Read more</a>
+          </p>
+        </div>
+        <div class="news-block">
+          <img
+            class="news-image"
+            src="./assets/luisa-brimble-Ikcn0alusdg-unsplash.jpg"
+            alt=""
+          />
+          <h4>Fall/Winter '25-26 Floral Trends</h4>
+          <p>
+            Explore the colors, textures, and bouquet styles set to define the
+            new season—straight from the fields to your home.
+            <a class="link-news" href="#">Read more</a>
+          </p>
+        </div>
+        <div class="news-block">
+          <img
+            class="news-image"
+            src="./assets/johanne-pold-jacobsen-o47uTPGoayA-unsplash.jpg"
+            alt=""
+          />
+          <h4>Vases that Make the Mood</h4>
+          <p>
+            Learn how the right vase transforms a simple bouquet into a design
+            statement.
+            <a class="link-news" href="#">Read more</a>
+          </p>
+        </div>
+        <div class="news-block">
+          <img
+            class="news-image"
+            src="./assets/irena-carpaccio-_6oPsp-XaIc-unsplash.jpg"
+            alt=""
+          />
+          <h4>Flower Care Made Simple</h4>
+          <p>
+            Easy, sustainable tips to keep your blooms fresher for
+            longer—without complicated routines.
+            <a class="link-news" href="#">Read more</a>
+          </p>
+        </div>
+      </div>
+    </article>
+
+    <!-- / News block -->
+
+    <!-- Contact block -->
+
+    <article id="contact-section" class="contact-container">
+      <h2 class="">Would you like to order something special?</h2>
+      <p>Please write us here and we will get back to you within 24 hours:</p>
+      <form
+        id="contact-form"
+        class="form-contact"
+        action="http://httpbin.org/anything"
+        method="post"
+      >
```

Comment:
Make sure to use the https method, as some browser will otherwise block this action